src/object.d

```diff
@@ -127,6 +127,68 @@ if (is(Obj : Object))
     assert(a <  "я");
 }
 
+void __move_post_blt(S)(ref S newLocation, ref S oldLocation) nothrow
```

Comment:
Please add a complete DDoc header to document this for other developers.

Reply:
Those of us that are trying to templatize the compiler<-->runtime interface plant to use the naming convention with a `_d_` prefix for runtime hooks.  The `__` prefix is reserved for compiler generated symbols and these aren't compiler-generated.  I realize it's splitting hairs somewhat because the compiler must be intimately aware of these symbols, but that's what we would prefer.  I realize that the `_d_` convention has not been followed well, and I plan to do something about that soon, but the `_d_` convention is what we've discussed and would prefer.

I don't expect you to change this PR to follow the `_d_` convention, though I wouldn't mind it either.  We have others to change, and we can change this with them when the time comes.  It's just an FYI to be aware of and to consider.

cc @wilzbach

Reply:
> I don't expect you to change this PR to follow the `_d_` convention, though I wouldn't mind it either. We have others to change, and we can change this with them when the time comes. It's just an FYI to be aware of and to consider.

I was aware of this when I implemented it, but I decided to stay as close as possible to the interface and implementation as defined by the DIP. I agree it should be changed to `_d_` in a later PR.

Reply:
Ddoc added.